src/core/js/models/questionModel.js

```diff
@@ -238,29 +238,35 @@ define([
 
         setupCorrectFeedback: function() {
 
+            var feedback = this.get("_feedback");
+
             this.set({
-                feedbackTitle: this.get('title'),
-                feedbackMessage: this.get("_feedback") ? this.get("_feedback").correct : ""
+                feedbackTitle: feedback ? feedback.title : "",
```

Comment:
```javascript
feedbackTitle: feedback.correctTitle || feedback.title ||  this.get('displayTitle') ||  this.get('title'),
```
^ would this work?

Reply:
Sure @moloko !
Thanks @oliverfoster !

Reply:
I don't think `correctTitle` is necessary but otherwise yes.

it's referenced enough times that it may be worth having a separate function like `setFeedback: function (message, graphic)` that handles adding in the appropriate title